app/Console/Command/ProcessDuplicatePlaces.php

```diff
@@ -65,11 +66,17 @@ public function configure(): void
             InputOption::VALUE_NONE,
             'Execute a dry-run of the process-duplicates script.'
         );
+        $this->addArgument(
+            'starting_cluster_id',
```

Comment:
Looking at other usages of `addArgument`, we always give them a `kebab-case` or `camelCase` name. While that's not consistent, I'd suggest to pick on of those to avoid adding a third convention